contingency/contingency-api/src/main/java/com/powsybl/contingency/json/IdentifierContingencyListSerializer.java

```diff
@@ -25,9 +25,9 @@ public IdentifierContingencyListSerializer() {
     @Override
     public void serialize(IdentifierContingencyList identifierContingencyList, JsonGenerator jsonGenerator, SerializerProvider serializerProvider) throws IOException {
         jsonGenerator.writeStartObject();
-        jsonGenerator.writeStringField("version", IdentifierContingencyList.getVersion());
         jsonGenerator.writeStringField("type", identifierContingencyList.getType());
         jsonGenerator.writeStringField("name", identifierContingencyList.getName());
+        jsonGenerator.writeStringField("version", IdentifierContingencyList.getVersion());
```

Comment:
shouldn't you put it before name, like all the other serializers?

Reply:
I needed it after attribut type but I can put it before name

Reply:
yes, it's better to put it at the same place for all